crmsh/qdevice.py

```diff
@@ -221,9 +297,7 @@ def init_db_on_qnetd(self):
         cmd = "corosync-qnetd-certutil -i"
         desc = "Step 1: Initialize database on {}".format(self.qnetd_addr)
         logger_utils.log_only_to_file(desc)
-        if self.askpass:
-            print(desc)
-        parallax.parallax_call([self.qnetd_addr], cmd, self.askpass)
+        parallax.parallax_call([self.qnetd_addr], cmd)
```

Comment:
I don't understand why we need to restart qnetd service here?

Reply:
Yes, the initial point is "start" to pickup the new nssdb.  

To extend it to "restart" is for the use case that I remove the nssdb directory manually while corosync-qnetd is still running.

Reply:
Still can not get your point here, why need to remove nssdb while qnetd is still running ?

Reply:
Just for advanced users/hackers/develpers' convenience sometimes. To remove the whole nssdb could simply clean up the whole nssdb, instead of the lengthy crmsh approach, or others.

Reply:
Suggest to keep this logic unchanged. Seems no harm here